Krivorotova Tatyana/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Cubes</title>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css?family=Lato:700&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <div class="weather-widget"><!-- Widget -->
+            <section class="weather-widget__city-card city-card-chicago"><!-- First card "Chicago" (img and name of the city) -->
```

Comment:
I think comments in html is excessive. In general, other developers should understand your markup by semantic (using suitable html tags, verbose class names, good naming for custom elements).

Reply:
Are class names that I've chosen correct and verbose enough? I mean, there was an issue with first version of names and I would like to know how good it is done now.

Reply:
I think it's good now 😄 - for now I pretty easily can understand what you do in this file)